spec/features/step_by_step_actions_spec.rb

```diff
@@ -85,6 +85,16 @@
     end
   end
 
+  context "Step by step has been published" do
+    scenario "show the relevant actions to the step by step author" do
+      given_there_is_a_published_step_by_step_page
+      when_I_visit_the_step_by_step_page
+      then_there_should_be_no_primary_action
+      and_there_should_be_no_secondary_action
+      and_there_should_be_tertiary_actions_to ["Unpublish", "View on GOV.UK"]
+    end
+  end
+
```

Comment:
It looks like this is also being tested in https://github.com/alphagov/collections-publisher/blob/master/spec/features/managing_step_by_step_pages_spec.rb#L369-L379.

Is there any way the duplication could be removed?

Reply:
Thanks, had meant to do that 👍 done.